src/compiler/program.ts

```diff
@@ -756,6 +934,7 @@ namespace ts {
                         diagnosticArgument = [fileName];
                     }
                     else if (!forEach(supportedExtensions, extension => findSourceFile(fileName + extension, toPath(fileName + extension, currentDirectory, getCanonicalFileName), isDefaultLib, refFile, refPos, refEnd))) {
+                        // (TODO: shkamat) Should this message be different given we support multiple extensions

```

Comment:
The problem is we do not know if the file exist but wrong extension or that it does not. may be we should add some elaboration to the error message. i would log a separate issue for it.